spark/content.md

```diff
@@ -0,0 +1,49 @@
+# What is Apache Spark™?
+
+Apache Spark™ is a multi-language engine for executing data engineering, data science, and machine learning on single-node machines or clusters. It provides high-level APIs in Scala, Java, Python, and R, and an optimized engine that supports general computation graphs for data analysis. It also supports a rich set of higher-level tools including Spark SQL for SQL and DataFrames, pandas API on Spark for pandas workloads, MLlib for machine learning, GraphX for graph processing, and Structured Streaming for stream processing.
+
+%%LOGO%%
+
+## Online Documentation
+You can find the latest Spark documentation, including a programming guide, on the [project web page](https://spark.apache.org/documentation.html). This README file only contains basic setup instructions.
+
+## Interactive Scala Shell
+
+The easiest way to start using Spark is through the Scala shell:
+
+```
+docker run -it spark /opt/spark/bin/spark-shell
+```
```

Comment:
The `markdownfmt` test will switch these fenced code blocks to tab indented code blocks because they have no language; set a language like `console` to keep them as fenced blocks.